### ISSUE TITLE
About the loss function 

### ISSUE THREAD

**Issue by DylanWusee** [xingyul/flownet3d#13]
Hi Xingyu,

Thank you for releasing code here. 

I noticed that the loss function used in the code is different from the one in the Paper. In the paper, you used a huber loss together with a cycle-consistency regularization.  

Are there particular reasons that you removed it from your code?

Thank you very much.
Best,
Dylan

**Comment by xingyul** [comment]
We provided huber loss in the code.

**Comment by DylanWusee** [comment]
But the huber loss in your code doesn't have the cycle-consistency regularization term. Is that correct?

**Comment by xingyul** [comment]
That's correct. Cycle-consistency regularization should be easy to implement.

**Comment by HimangiM** [comment]
I am having some issues in implementing it. It would be great if you can upload it.
